utils/release/npm-publish-package.mjs

```diff
@@ -134,7 +146,8 @@ async function updateWorkspaceDependent(version) {
   )) {
     if (
       dependencies.find(
-        (dependency) => dependency.target === dependencyPackageName
+        (/** @type {{target:string}} **/ dependency) =>
+          dependency.target === dependencyPackageName
```

Comment:
Would using typecast `as string` work in this case instead?

Reply:
No, because we're in JS.

Reply:
Yes right 🤦